R/extract_parameters.R

```diff
@@ -229,6 +206,26 @@
   }
 
 
+
+  # ==== Std Coefficients for other methods than "refit"
+
+  if (!is.null(standardize)) {
+    # give minimal attributes required for standardization
+    temp_pars <- parameters
+    class(temp_pars) <- c("parameters_model", class(temp_pars))
+    attr(temp_pars, "ci") <- ci
+    attr(temp_pars, "obj_name") <- model # pass the model as is (this is a cheat - teehee!)
+
+    std_parms <- effectsize::standardize_parameters(temp_pars, method = standardize)
+    parameters$Coefficient <- std_parms$Std_Coefficient
```

Comment:
You asked why the column is named "Coefficient", but you explicitely did this here?

Reply:
Fixed!